lib/view_component/collection_iteration.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module ViewComponent # :nodoc:
```

Comment:
Worth a comment that this is the same as (copied from) [`PartialIteration`](https://github.com/rails/rails/blob/ab18f60841c11427948be444ee5ff0008fd4c3fb/actionview/lib/action_view/renderer/collection_renderer.rb#L6)?